{
 "ETag": "W/\"7de1f7a1252763cf7bca779ec9e1c60e769dcf8c76e7f86d98bd29cbc32cc73f\"",
 "Last-Modified": "Wed, 10 Apr 2024 13:15:22 GMT",
 "base": {
  "repo": {
   "name": "pyctest-feedstock"
  }
 },
 "closed_at": null,
 "created_at": "2023-09-28T01:21:48Z",
 "draft": false,
 "head": {
  "ref": "rebuild-python312-0-1_h0c10b2"
 },
 "html_url": "https://github.com/conda-forge/pyctest-feedstock/pull/23",
 "id": 1533323020,
 "labels": [
  {
   "color": "771617",
   "default": false,
   "description": "",
   "id": 6021975541,
   "name": "py312-compiler-error",
   "node_id": "LA_kwDOCaN-gM8AAAABZvAN9Q",
   "url": "https://api.github.com/repos/conda-forge/pyctest-feedstock/labels/py312-compiler-error"
  }
 ],
 "mergeable": true,
 "mergeable_state": "unstable",
 "merged": false,
 "merged_at": null,
 "number": 23,
 "state": "open",
 "updated_at": "2023-09-29T07:53:46Z"
}